app/javascript/crayons/Buttons/Button.jsx

```diff
@@ -0,0 +1,62 @@
+import { h } from 'preact';
+import { useState } from 'preact/hooks';
+import classNames from 'classnames/bind';
+import { Icon } from '@crayons';
+
+export const ButtonNew = (props) => {
```

Comment:
I've renamed it for now to not confuse ppl and make it easie to import it from `@crayons` since we already have "Button" component.

Reply:
When you import, we can do this for now to have less refactoring later.

`import { NewButton: Button } from '@crayons'`

Reply:
ohhh, neeeeeat! I'll do that!